playbooks/roles/cyclecloud_cluster/projects/common/cluster-init/files/centos/init_apptainer.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+yum install https://github.com/apptainer/singularity/releases/download/v3.8.7/singularity-3.8.7-1.x86_64.rpm -y
+yum install https://github.com/apptainer/apptainer/releases/download/v1.1.6/apptainer-1.1.6-1.x86_64.rpm -y
```

Comment:
What is the purpose of installing both singularity from Feb 2022 release and Apptainer from Feb 2023 release? Are both required? My understanding is that Apptainer is the new name, but is there a dependency here?

Reply:
@JingchaoZhang would be best to answer.

Reply:
> 

Direct install of apptainer on CentOS 7.9 will result in dependence failures. I just found out a more lightweight solution and will update the script shortly.

Reply:
Thank you for looking at my comment and testing out something lighter weight! Is this additional package only required for CentOS 7.9? Maybe this should be an issue on Apptainer too so that the correct dependencies are pulled in more seamlessly.